go/common/crypto/signature/signature.go

```diff
@@ -666,3 +666,14 @@ func BuildPublicKeyBlacklist(allowTestKeys bool) {
 		blacklistedPublicKeys.Store(pk, true)
 	}
 }
+
+// NewBlacklistedKey returns the PublicKey from the given hex or panics.
+// The given key is also added to the blacklist.
+func NewBlacklistedKey(hex string) PublicKey {
+	var pk PublicKey
+	if err := pk.UnmarshalHex(hex); err != nil {
+		panic(err)
+	}
+	blacklistedPublicKeys.Store(pk, true)
```

Comment:
@Yawning suggested to panic in case the key already exists -- to catch ID reuse. This is a good idea, also in this case you probably need to change the IDs to use something unique as those small order points are already blacklisted.

Reply:
Good idea!  I've implemented it and changed the IDs.  The new IDs have a prefix of `1abe11ed` (for "labelled") -- **c**ommon p**ool** continues with `c00l` and **fee** **acc**umulator continues with `feeacc` :)